src/common/transformations/src/transformations/mark_decompression_convert_constant_folding.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/mark_decompression_convert_constant_folding.hpp"
+
```

Comment:
Please consider to place fp16 compression transformations and related sources into the directory `fp16_compression` like it is done for Transpose Sinking